core/src/Cuda/Kokkos_Cuda_KernelLaunch.hpp

```diff
@@ -293,7 +293,7 @@ struct DeduceCudaLaunchMechanism {
   // UseT<F<KAL   LCG LCG C  C        LCG  LG C  L    LCG  CG C  C
   // Kal<F<CMU     CG LCG C  C         CG  LG C  G     CG  CG C  C
   // CMU<F          G LCG G  G          G  LG G  G      G  CG G  G
-  static constexpr const Experimental::CudaLaunchMechanism launch_mechanism =
+  static constexpr Experimental::CudaLaunchMechanism launch_mechanism =
```

Comment:
Although you are right in that the const qualifier is superfluous, please undo that change.

Reply:
Yeah sorry I forgot to revert that... and done !